### PR TITLE
Update parameter-types

### PR DESCRIPTION
**Pull request by tkatzir**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Update the parameter-types page.

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@glicht @anara123 Doc review completed.

File: docs/integrations/parameter-types.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ See [here](https://xsoar.pan.dev/docs/integrations/yaml-file#configuration) for
 
 This type of parameter creates a checkbox in the integration configuration. When the checkbox is ticked, the value in
 the integration code is `True`, and `False` otherwise.
-The type number is: 0.
+The type number is: 8.
```
Comment (anara123) [inline docs/integrations/parameter-types.md:15]: Boolean is 8 and short text is 8?

**Review by anara123 — APPROVED** [review]
See my comment

File: docs/integrations/parameter-types.md (hunk 1 above)
Comment (tkatzir) [reply docs/integrations/parameter-types.md:15]: Boolean is 8, and Short Text is 0.
[Fixed](https://github.com/demisto/content-docs/pull/1393/commits/873142660357adc8e046244e73e716bc61a604f8).

**Pull request merged**